Basic-Car-Maintenance/Shared/Settings/ViewModels/SettingsViewModel.swift

```diff
@@ -13,11 +13,13 @@ import Foundation
 final class SettingsViewModel {
     let authenticationViewModel: AuthenticationViewModel
     var contributors: [Contributor]?
+  var chooseAppIconViewModel: ChooseAppIconViewModel
```

Comment:
Is this needed? It's not being used in the `SettingsViewModel`
I think it's fine to pass it directly when you're in the `SettingsView`

Reply:
Done.